co2calculator/distances.py

```diff
@@ -34,7 +34,7 @@
 class StructuredLocation(BaseModel, extra=Extra.forbid):
     address: Optional[str]
     locality: str
-    country: CountryCode
+    country: Union[CountryCode, CountryName]
```

Comment:
You did not add the country name to the class TrainStation because the trainline EU database only supports country codes, right?

Reply:
one more thing: if I saw correctly, the trainline EU database only has the two-letter (alpha-2) country codes. So could it be that the train station class requires a different country code class that only allows alpha-2 instead of alpha-2 and alpha-3?

Reply:
one more thing: if I saw correctly, the trainline EU database only has the two-letter (alpha-2) country codes. So could it be that the train station class requires a different country code class that only allows alpha-2 instead of alpha-2 and alpha-3?

Reply:
Thanks for noticing, I added a commit that is restricting it to 2 letter country codes for the train stations